staging/src/k8s.io/apiserver/pkg/util/proxy/proxy.go

```diff
@@ -78,7 +78,7 @@ func ResolveEndpoint(services listersv1.ServiceLister, endpoints listersv1.Endpo
 			continue
 		}
 		for i := range ss.Ports {
-			if ss.Ports[i].Name == svcPort.Name {
+			if ss.Ports[i].Port == svcPort.Port {
```

Comment:
if this works, can we get rid of the svcPort var and just use 443?

Reply:
should be `targetPort` here

Reply:
`targetPort ` is not valid here, because is maybe a string.

Reply:
@sttts  first commit removed